### PR TITLE
Generate cli docs in hugo-style markdown, to include them in the main…

### PR DESCRIPTION
**Pull request by fekete-robert**
… banzaicloud doc site

| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     | no|yes
| Deprecations?   | no|yes
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | Apache 2.0

### What's in this PR?
Changes the way the automated command-line docs are generated, so the docs can be incorporated into the main banzaicloud.com doc site.

### Why?
So the CLI docs can be included in the main doc site in HTML format.

### CODE REVIEW

File: cmd/docs/generate.go [hunk 1]
```diff
@@ -15,16 +15,41 @@
...
 
 func main() {
 	cmd.Init("", "", "", "")
+
+	const fmTemplate = `---
+title: %s
+slug: %s
+---
+
+> This file was generated automatically. Do not modify it.
```
Comment (orymate) [inline cmd/docs/generate.go]: why is this needed? do we want it to be shown in the rendered docs?

Comment (fekete-robert) [reply cmd/docs/generate.go]: Yes, so noone starts editing something in the md file, because it will be automatically lost

Comment (orymate) [reply cmd/docs/generate.go]: but how is this information relevant for the readers of the rendered site?
